### DISCUSSION TITLE
Another "The media could not be loaded..." issue

### DISCUSSION THREAD

**Discussion by mechpanos** [blakeblackshear/frigate#7795] [General]
Hi all! I have read tons about this issue, tried several solutions, with no result. 
The problem in my situation started occuring when I moved my HA installation from bare metal, to Proxmox. 
I have a Coral accelerator, but no more HW acceleration, since I can't passthrough my gpu to HA vm, it is occupied by the proxmox OS.
However, the mysterious part is that some clips play OK, and after I play them they dissapear.
(Frigate installed as HA addon.)
Any ideas?

My config:
https://pastebin.com/fkAzdMQW

I have some errors in my logs, but can't figure much, any help?
https://pastebin.com/9CU4Fbu8

**Comment by NickM-27** [comment]
looks like a lot of bad data coming from the camera

also `Detection appears to be stuck. Restarting detection process...` seems like proxmox is causing issues with the coral

**Reply by mechpanos** [reply]
Thnx for the comment. Yes TP Link camera is wi-fi and sometimes has bad signal, but Hikvision doorbell is ethernet and has the same issue, too. About proxmox + coral, is there something I could do? Maybe add the coral to a powered usb hub instead of a usb port?...or is it just software, between proxmox + coral...I 'll see proxmox logs, as well...

**Reply by NickM-27** [reply]
assuming it is the same hardware, it is likely a problem with proxmox itself. Some users have said that using one type of install (can't remember if it was lxc or vm) works better than the other for this.
